img_up/models.py

```diff
@@ -0,0 +1,19 @@
+from django.db import models
+import uuid
+from django.contrib.auth.models import User
+
+
+# Create your models here.
+class Epilepsy(models.Model):
+    id = models.UUIDField(primary_key=True, default=uuid.uuid4, editable=False)
+    admin_name = models.CharField(max_length=30)
+    state = models.CharField(max_length=30)
+    numOfTotalCases = models.PositiveSmallIntegerField()
+    numOfChildCases = models.PositiveSmallIntegerField()
+    numOfAdultCases = models.PositiveSmallIntegerField()
+    created_at = models.DateTimeField(auto_now_add=True)
+    updated_at = models.DateTimeField(auto_now=True)
+
+class EpilepsyUserInput(Epilepsy):
+    user = models.ForeignKey(User, default=uuid.uuid4, on_delete=models.CASCADE)
```

Comment:
There shouldn't be a `default` argument for the `ForeignKey`; the foreign key should point to an existing primary key in the User table, so by using `default`, it's possible the user might have a key that points to no user in the User table, resulting in errors.